README.md

```diff
@@ -29,7 +29,7 @@ Kyo is available on Maven Central in multiple modules:
 
 For Scala 3:
 
-```scala 
```

Comment:
I think we need these spaces here. There's this hack to allow code examples that aren't compiled: https://github.com/getkyo/kyo/blob/d41f50e8a41f524cf7aaf1cbbc3aeca54300772e/build.sbt#L388. We should address this hack, though.

Reply:
Ah, got it... sorry. Will revert.